rclpy/rclpy/qos.py

```diff
@@ -285,13 +288,18 @@ class QoSReliabilityPolicy(QoSPolicyEnum):
     RMW_QOS_POLICY_RELIABILITY_BEST_EFFORT = 2
     BEST_EFFORT = RMW_QOS_POLICY_RELIABILITY_BEST_EFFORT
 
+# Naming the class ReliabilityPolicy QoSReliabilityPolicy 
+# because it is used in other places with the "Qos" infront
+QoSReliabilityPolicy=ReliabilityPolicy
 
-class QoSDurabilityPolicy(QoSPolicyEnum):
+
+
```

Comment:
Looks like there's 3 blank lines, the linter will probably complain that it should only be 2 blank lines between top level definitions